app/design/frontend/Magento/elogic-theme/Magento_Cms/web/css/source/_module.less

```diff
@@ -181,42 +183,43 @@
     left: 0;
 }
 
+//              SALE
+//________________________________________
 
+.sale {
+	.lib-vendor-prefix-grid-display(grid);
+	.lib-vendor-prefix-grid-template-columns(1fr 1fr);
+	.lib-vendor-box-align(center);
+    .lib-css(background, url('https://i.ibb.co/J2T8NXR/image.png'));
+    // background: linear-gradient( rgba(0, 0, 0, 0.05), rgba(0, 0, 0, 0.05) ), url('https://i.ibb.co/BCM35Lm/bg.jpg');
+	background-size: cover;
+	margin: 30px 0;
+}
 
+.sale-button {
+	a {
+		text-decoration: 0;
+		border-bottom: 1px solid @color-elogic-black;
+		padding-bottom: 5px;
+		transition: all 0.3s ease;
+		-webkit-transition: all 0.3s ease;
+		-moz-transition: all 0.3s ease;
+		-ms-transition: all 0.3s ease;
+		-o-transition: all 0.3s ease;
```

Comment:
you don't need vendor prefixes for the `transition`. It's supported by all browsers